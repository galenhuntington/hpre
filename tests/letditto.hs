foo :: Int -> Bool
''  0 = not $ let once True = let twice False = False
                                  ''    _     = True
                              in once $ twice True
                  ''   _    = False
              in once True
''  x = True
