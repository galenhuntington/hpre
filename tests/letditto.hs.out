foo :: Int -> Bool
foo{-#COLUMN 5#-}0 = not $ let once True = let twice False = False
                                  twice{-#COLUMN 41#-}_     = True
                              in once $ twice True
                  once{-#COLUMN 24#-}_    = False
              in once True
foo{-#COLUMN 5#-}x = True
